src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -236,7 +236,6 @@ async ValueTask<int> ReadAheadWithZeroByteReadAsync()
 
         private bool CheckKeepAliveTimeoutExceeded()
         {
-            // We only honor a Keep-Alive timeout on HTTP/1.0 responses.
```

Comment:
Maybe leave a comment about honoring this for HTTP/1.1 even though it isn't part of the HTTP/1.1 spec.